packages/protocol/test/governance/bondeddeposits.ts

```diff
@@ -79,6 +79,37 @@ contract('LockedGold', (accounts: string[]) => {
     await lockedGold.createAccount()
   })
 
+  describe('#isAccount()', () => {
+    it('created account should exist', async () => {
+      const b = await lockedGold.isAccount(account)
+      assert.equal(b, true)
+    })
+    it('account that was not created should not exist', async () => {
+      const b = await lockedGold.isAccount(accounts[2])
+      assert.equal(b, false)
+    })
+  })
+
+  describe('#isDelegate()', () => {
+    const delegate = accounts[1]
+    let sig
```

Comment:
Nit: since this isn't used in the `it`s, could be declared as a `const` within the `beforeEach`.